GliaWidgets/Sources/ViewController/Chat/ChatViewController.swift

```diff
@@ -124,7 +124,7 @@ final class ChatViewController: EngagementViewController, PopoverPresenter {
             case .setMessageText(let text):
                 view?.messageEntryView.messageText = text
             case .sendButtonHidden(let hidden):
```

Comment:
Does it make sense to rename `.sendButtonHidden` also?

Reply:
Good point f2aad2984a8e5f46ae00b659a95490ea27254cfa.